src/xenia/memory.cc

```diff
@@ -909,6 +900,7 @@ bool BaseHeap::AllocFixed(uint32_t base_address, uint32_t size,
     page_entry.allocation_protect = protect;
     page_entry.current_protect = protect;
     page_entry.state = kMemoryAllocationReserve | allocation_type;
+    unreserved_page_count_--;
```

Comment:
This shouldn't be done if the page is already reserved (and the game is just committing it, as a common usage case).